saltpylint/pep8.py

```diff
@@ -417,7 +417,7 @@ def register(linter):
     '''
     required method to auto register this checker
     '''
-    if HAS_PEP8 is False:
+    if HAS_PYCODESTYLE is False:
```

Comment:
But of course! As there weren't any tests available here I built this locally and tested it against large pieces of the `salt` project and it worked as expected. There were slight differences in the lint output likely caused by patches given the version of `pylint` being used is about 6 months old.

The `pep8` library was simply renamed `pycodestyle` a year or so ago however the underlying code is the same.